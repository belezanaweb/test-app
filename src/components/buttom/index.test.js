import 'jest-styled-components';
// do not forget the /native at the end of the line.
import React from 'react';
import renderer from 'react-test-renderer';
import ButtomProject from './index';

describe('Component Buttom Snapshot', () => {
	test('Buttom Snapshot', () => {
		const tree = renderer.create(<ButtomProject name="Leo" background="#fff" />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
