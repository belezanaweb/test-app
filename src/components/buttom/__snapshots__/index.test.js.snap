// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Buttom Snapshot Buttom Snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "transparent",
      "borderRadius": 10,
      "borderWidth": 2,
      "marginBottom": 0,
      "marginLeft": 0,
      "marginRight": 0,
      "marginTop": 0,
      "opacity": 1,
      "paddingBottom": 10,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 10,
      "width": "100%",
    }
  }
  testID="input-component"
>
  <Text
    background="#fff"
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "Roboto-Bold",
          "fontSize": 14,
          "lineHeight": 16,
          "textAlign": "center",
          "textTransform": "uppercase",
        },
      ]
    }
  >
    Leo
  </Text>
</View>
`;
